src/FileConventions/Library.fs

```diff
@@ -60,3 +60,16 @@ let DetectAsteriskInPackageReferenceItems(fileInfo: FileInfo) =
         )
 
     asteriskInPackageReference.IsMatch fileText
+
+let DetectMissingVersionsInNugetPackageReferences(fileInfo: FileInfo) =
+    assert (fileInfo.FullName.EndsWith(".fsx"))
+    use streamReader = new StreamReader(fileInfo.FullName)
+    let fileText = streamReader.ReadToEnd()
+
+    let versionsInNugetPackageReferences =
+        Regex(

```

Comment:
@realmarv I'm sorry, this logic is unnecesarily complicated; remember the rule of "do the simplest thing to make the test pass", and to distinguish `#r "nuget:` lines that have version or not you can just check if there is a comma in the line, without the need to use Regex

Reply:
Done